tests/test_pytest/reader_test.py

```diff
@@ -0,0 +1,75 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+"""Tests all the readers given input test files."""
+
+import pytest
+from glob import glob
+from geoips.commandline.log_setup import setup_logging
+from geoips.interfaces import readers
+
+LOG = setup_logging()
+
+
+@pytest.mark.parametrize(
```

Comment:
@AJThorpe I think maybe we want to add a pytest for a reader in a different repo, so we can see how we want to connect them together?  I think geoips_clavrx has a reader that could be used as  a test case.

Reply:
From Evans comment I think we should group with this package  in a sperate sub directory and then maybe have a separate reader test for the clavrx reader?

Reply:
Yeah!  The geoips_clavrx reader test would have to go in the geoips_clavrx repo.  See if you can get that set up ?

Reply:
Geoips clavrx repo should be updated to work with this new layout!

Reply:
Clavrx repo is updated @mindyls !